Loop/Managers/DeviceDataManager.swift

```diff
@@ -376,6 +376,14 @@ final class DeviceDataManager {
         return Manager.init(rawState: rawState) as? CGMManagerUI
     }
     
+    func checkDeliveryUncertaintyState() {
+        if let pumpManager = pumpManager, pumpManager.status.deliveryIsUncertain {
+            DispatchQueue.main.async {
+                self.deliveryUncertaintyAlertManager!.showAlert()
```

Comment:
[nit] now that this code moved here consider changing

```swift
                self.deliveryUncertaintyAlertManager!.showAlert()
```

to

```swift
                self.deliveryUncertaintyAlertManager?.showAlert()
```

...if only to discourage force-unwraps unless absolutely necessary

Reply:
This one is intentional. `deliveryUncertaintyAlertManager` should only be nil if we do not have a pumpManager.  If we do have a pumpManager, and deliveryStateIsUncertain = true, and we don't have a deliveryUncertaintyAlertManager, that's a failed assertion that should stop the program.